tests/functional/conftest.py

```diff
@@ -32,15 +32,15 @@ def app(pyramid_app, db_engine):
     return TestApp(pyramid_app)
 
 
-@pytest.yield_fixture(scope='session')
+@pytest.fixture
```

Comment:
The change of scope here causes the functional tests to fail because `db_engine` is now created once per test function instead of once for the whole session.

The functional tests are only run on CI builds by our Jenkins server, not Travis, which unfortunately are only visible to people at Hypothesis. You can run them locally using `tox -e functional`.